docs/samples/tensorrt/README.md

```diff
@@ -0,0 +1,65 @@
+
+# Predict on a KFService using TensorRT Inference Server
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. Your cluster's Istio Egresss gateway must [allow Google Cloud Storage](https://knative.dev/docs/serving/outbound-network-access/)
+
+
+> Knative is not able to resolve containers from the NVIDIA container registry.
+To work around this you need to skip resolution for nvcr.io by editing knative config-deployment.yaml. See details here: https://github.com/knative/serving/issues/4355
+## Create the KFService
+Apply the CRD
+```
+kubectl apply -f tensorrt.yaml 
+```
+
+Expected Output
+```
+kfservice.serving.kubeflow.org/simple-string created
```

Comment:
We've been naming these samples framework-dataset

Reply:
This is the name of the nvidia example, how about i call it tensorrt-simple-string?